sycl/include/sycl/aspects.hpp

```diff
@@ -12,47 +12,18 @@
 namespace sycl {
 __SYCL_INLINE_VER_NAMESPACE(_V1) {
 
+#define __SYCL_ASPECT(ASPECT, ID) ASPECT = ID,
+#define __SYCL_ASPECT_DEPRECATED(ASPECT, ID, MESSAGE)                          \
+  ASPECT __SYCL2020_DEPRECATED(MESSAGE) = ID,
+#define __SYCL_SPECIAL_ASPECT_DEPRECATED(ASPECT, ID, MESSAGE)                  \
```

Comment:
I'd rename to `__SYCL_ASPECT_DEPRECATED_ALIAS`.

Reply:
Done 3002a43